src/core/test/base/build.sbt

```diff
@@ -1 +1 @@
-// nothing here
+//> DependsOn: core-env
```

Comment:
This comes from the serialization commit, which is a separate PR but this pr depends on it. In short, i needed some file utilities in the roundTripTestBase class